sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/SdkMeterProviderBuilder.java

```diff
@@ -74,7 +74,8 @@ public SdkMeterProviderBuilder setResource(@Nonnull Resource resource) {
    *   .setAggregatorFactory(aggregatorFactory).build());
    * }</pre>
    */
-  public SdkMeterProviderBuilder registerView(InstrumentSelector selector, View view) {
+  public SdkMeterProviderBuilder registerView(
+      @Nonnull InstrumentSelector selector, @Nonnull View view) {
```

Comment:
is this necessary? I thought we had `@ParametersAreNonNullByDefault` everywhere.